src/services/update.js

```diff
@@ -0,0 +1,30 @@
+const sortTodo = (todos) => todos.sort((a, b) => a.index - b.index);
+export const save = (todos) => localStorage.setItem('todos', JSON.stringify(todos));
+
+export const saveToStorage = (todos) => {
+  const sortedTodos = sortTodo(todos);
+  save(sortedTodos);
+};
+
+export const orderAndSave = (todos) => {
+  const orderedTodos = todos.map((todo, index) => {
+    todo.index = index + 1;
+    return todo;
+  });
+  save(orderedTodos);
+};
+
+export const updateStatus = (e, todos) => {
+  const index = e.target.id.slice(-1);
+  todos[index - 1].completed = !todos[index - 1].completed;
+  saveToStorage(todos);
+  return todos;
+};
+
+export const getTodos = () => {
+  const todos = localStorage.getItem('todos');
+  if (todos) {
+    return JSON.parse(todos);
+  }
+  return [];
+};
```

Comment:
- [ ] Your method for storing the array of object and sorting it, is good but you still more to make it work.

![jjdj](https://user-images.githubusercontent.com/61562626/134659067-8e6d0b8a-bef1-4fe0-bade-bfa4a1c983ce.jpeg)